### PR TITLE
grub-editenv: add page

### PR DESCRIPTION
**Pull request by SAYUK09**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by Magrid0 — CHANGES_REQUESTED** [review]
Hi, thanks for contributing to this project, but unfortunately we can't accept this pull request, we advise to take a look to the [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md) and update the file.

**Comment by SAYUK09** [comment]
> Hi, thanks for contributing to this project, but unfortunately we can't accept this pull request, we advise to take a look to the [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md) and update the file.

I have made the changes. Please take a look now.
Thanks @Magrid0

File: pages/linux/grub-editenv.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+- Reset the `saved_entry` variable to the default:
+
+  `grub-editenv /boot/grub/grubenv unset saved_entry`
+
+- Append "quiet splash" to the kernel command line:
+
+  `grub-editenv /boot/grub/grubenv list kernel_cmdline`
+
+## See Also
```
Comment (Magrid0) [inline pages/linux/grub-editenv.md:21]:
```suggestion

```

File: pages/linux/grub-editenv.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- More information: [GRUB Documentation](https://www.gnu.org/software/grub/manual/grub/grub.html).
```
Comment (Magrid0) [inline pages/linux/grub-editenv.md:22]:
```suggestion
```

File: pages/linux/grub-editenv.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# grub-editenv
+
+> Edit GRUB environment variables on Linux.
```
Comment (Magrid0) [inline pages/linux/grub-editenv.md:3]:
```suggestion
> Edit GRUB environment variables on Linux.
> More information: <https://www.gnu.org/software/grub/manual/grub/grub.html>.

```

**Review by Magrid0 — CHANGES_REQUESTED** [review]
it's still not correct

File: pages/linux/grub-editenv.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+> More information: <https://www.gnu.org/software/grub/manual/grub/grub.html>.
+
```
Comment (Magrid0) [inline pages/linux/grub-editenv.md:5]:
```suggestion
```

File: pages/linux/grub-editenv.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+- Set the default boot entry to "Ubuntu" (Assuming the boot entry exists):
+
+  `grub-editenv /boot/grub/grubenv set default=Ubuntu`
```
Comment (Magrid0) [inline pages/linux/grub-editenv.md:9]:
```suggestion
`grub-editenv /boot/grub/grubenv set default=Ubuntu`
```

File: pages/linux/grub-editenv.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+- Display the current value of the `timeout` variable:
+
+  `grub-editenv /boot/grub/grubenv list timeout`
```
Comment (Magrid0) [inline pages/linux/grub-editenv.md:13]:
```suggestion
`grub-editenv /boot/grub/grubenv list timeout`
```

File: pages/linux/grub-editenv.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+- Reset the `saved_entry` variable to the default:
+
+  `grub-editenv /boot/grub/grubenv unset saved_entry`
```
Comment (Magrid0) [inline pages/linux/grub-editenv.md:17]:
```suggestion
`grub-editenv /boot/grub/grubenv unset saved_entry`
```

File: pages/linux/grub-editenv.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+- Append "quiet splash" to the kernel command line:
+
+  `grub-editenv /boot/grub/grubenv list kernel_cmdline`
```
Comment (Magrid0) [inline pages/linux/grub-editenv.md:21]:
```suggestion
`grub-editenv /boot/grub/grubenv list kernel_cmdline`
```

**Review by Magrid0 — CHANGES_REQUESTED** [review]
with this last change it should pass the check

**Comment by SAYUK09** [comment]
Thanks for helping our @kbdharun

**Review by kbdharun — APPROVED** [review]
LGTM, Welcome to tldr and thanks for your contribution.

**Review by Magrid0 — APPROVED** [review]
Looks good to me, thanks for your contribution!

**Pull request merged**